Formalist/TextEditorConfiguration.swift

```diff
@@ -26,6 +26,11 @@ public struct TextEditorConfiguration {
         case returnKey
     }
 
+    public enum DoneButtonStyle {
+        case custom(title: String)
+        case builtIn
```

Comment:
Did you consider `standard` or `default`? I feel it's more in-line with other Swift APIs.

Reply:
I had switched to `builtIn` when the syntax highlighter misidentified `default` as a reserved keyword 😆 but everywhere else in the app uses `.default`. I'll switch over